### PR TITLE
Feature/updating recipe cards

### PR DESCRIPTION
**Pull request by jeremiahblackol**
# PR Description
Please include a summary of the change:

This change makes it so add to recipe functionality works on buttons.

Commit Emojis:

:tada: Initial commit
:construction: HTML modifications/updates.
:art: CSS modifications/updates.
:crystal_ball: JS modifications/updates.
:lipstick: hotfix any code.
:bar_chart: new tests
:camera_flash: adding screenshots
:bento: Adding new assets or images
:beetle: If it is a bug or issue fix, describe the issue that was fixed.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [ ] My code follows the Turing style guidelines
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/recipe.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ class Recipe {
     this.instructions = recipe.instructions;
     this.tags = recipe.tags;
     this.ingredientsData = ingredientsData;
+    this.isFavorite = false;
```
Comment (Billwilke42) [inline src/recipe.js:9]: Really like that you added a boolean value for this!

File: src/scripts.js [hunk 2]
```diff
@@ -106,6 +108,43 @@ function viewFavorites() {
   }
 }
 
+function viewRecipesToCook() {
```
Comment (Billwilke42) [inline src/scripts.js:111]: Great work on this method!

File: src/scripts.js [hunk 3]
```diff
@@ -187,14 +265,13 @@ function populateCards(recipes) {
     cardArea.classList.remove('all')
   }
   recipes.forEach(recipe => {
+    let recipeToCook = recipe.isRecipeToCook ? "-" : "+"
```
Comment (Melizzo) [inline src/scripts.js:268]: Great Idea to utilize the inner text for the changing of the button.

File: src/recipe.js (hunk 1 above)
Comment (Melizzo) [reply src/recipe.js:9]: Agreed - this REALLY helped in the coding.

**Pull request merged**